os/sdk-modifying-coreos.md

```diff
@@ -28,7 +27,40 @@ git config --global user.name "Your Name"
 
 **NOTE**: Do the git configuration as a normal user and not with sudo.
 
-### Install repo
+### Option 1: Using Cork
+
+The `cork` utility, included in the CoreOS [mantle](https://github.com/coreos/mantle) project, can be used to create and work with an SDK chroot.
+
+In order to use this utility, you must additionally have the [golang](https://golang.org/) toolchain installed and configured correctly.
+
+First, install the cork utility:
+
+```sh
+git clone https://github.com/coreos/mantle
+cd mantle
+./build cork
+mkdir ~/bin
+mv ./bin/cork ~/bin
+export PATH=$PATH:$HOME/bin
+```
+
+You may want to add the `PATH` export to your shell profile (e.g. `.bashrc`).
+
+
+Next, the cork utility can be used to create an SDK chroot:
+
+```sh
+mkdir coreos-sdk
+cd coreos-sdk
+cork create --sdk-version=master
+cork enter
+```
+
+To use the SDK chroot in the future, run `cork enter` from the above directory.
+
+### Option 2: Manually
```

Comment:
I wouldn't exactly call this manual, it is more of just changing the order in which the SDK is set up. Maybe call it "Using repo and cros_sdk"